### ISSUE TITLE
Tb78 regression: navigation of advanced settings tabs with cursor keys broken

### ISSUE THREAD

**Issue by RealRaven2000** [RealRaven2000/SmartTemplate4#95]
It should be possible to navigate between the different tabs of the advanced section of SmartTemplates Settings, using keyboard only (cursor keys)

![image](https://user-images.githubusercontent.com/6314674/99396597-d74b9080-28d9-11eb-956d-80aefc7b1e19.png)

This had been working in Thunderbird 68, but is now broken in Thundebird 78
